Shaders/HighPassSharpen.fx

```diff
@@ -14,163 +13,112 @@ uniform float HighPassSharpOffset <
 	ui_type = "drag";
 	ui_min = 0.00; ui_max = 1.00;
 	ui_tooltip = "Additional adjustment for the blur radius. Values less than 1.00 will reduce the radius limiting the sharpening to finer details.";
-	ui_step = 0.20;
 > = 1.00;
 
 uniform int HighPassBlendMode <
```

Comment:
I agree, that is much better to work with. I made use of the combo box in the Clarity shader as well.